README.md

```diff
@@ -40,8 +40,8 @@ Copy and paste the following snippet into your build .yml file:
 Additional arguments:
 
 | CLI                       | Action input            | Default |
-| ------------------------- | ----------------------- | ------- |
-| --fail-on WARN            | fail-on-diff            | true    |
+|---------------------------|-------------------------|---------|
+| --fail-on                 | fail-on                 | 'ERR'   |
```

Comment:
The default oasdiff exit code is 0 if there is a breaking changes, why do you want to change that?